### PR TITLE
Add pipedream extension

### PR DESCRIPTION
**Pull request by olekristianbe**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/pipedream/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Quick actions to activate/deactivate workflows
+- Error tracking and easy access to error logs
+- Sort and filter workflows based on various criteria
+
+## Setup
+
+1. Install the extension from the Raycast store.
+2. You'll need a Pipedream API key to use this extension. To get one:
+   - Log in to your Pipedream account
+   - Go to [Pipedream Account Settings](https://pipedream.com/settings/account)
```
Comment (mil3na) [inline extensions/pipedream/README.md:19]: This link doesn't go to the Account section. Either fix it or tell the user in text.

File: extensions/pipedream/src/hooks/useUserInfo.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+export function useUserInfo(): UseUserInfoReturn {
+  const {
+    data: userInfo,
+    isLoading,
+    error,
+    revalidate,
+  } = useCachedPromise(
+    async () => {
+      try {
+        console.log("Fetching user info...");
```
Comment (mil3na) [inline extensions/pipedream/src/hooks/useUserInfo.ts:26]: If not needed, please remove the console.logs

**Review by mil3na — CHANGES_REQUESTED** [review]
Hi!
Thank you for your contribution! My name is Milena and I will be reviewing it today.

I had some issues using your extension and also left a few comments on the code. Please, take a look and tell me what you think:

- I guess I entered the wrong API Key on the first time and I got these errors. Do you mind making them more readable?
CleanShot 2024-07-19 at 15 26 29
CleanShot 2024-07-19 at 15 26 53

- On this screen, you could add an info tooltip, instead of put the information on the body. Same for input validation.
CleanShot 2024-07-19 at 15 27 56

- I tried to create a workflow, but I could not find the id you mention on the form. Should something be created on pipedream first?

**Comment by olekristianbe** [comment]
Hi,

Thank you fast reply! I will make the following improvements:

- Improve the error message.
- Include instructions on how to change the API Key for users who enter an incorrect one.
- Implement the use of info props in the "Add Workflow" command.
- Rename the "Add Workflow" command to "Connect Workflow" or a similar term to clarify its purpose of managing existing workflows in Pipedream.
- Update the incorrect link in the README.
- Remove the console log used for testing.

I appreciate your feedback and will implement these changes. I'll update you once completed. Have a great day!

**Comment by mil3na** [comment]
Hi! 

Thanks for the updates. I still see a few issues:
- Changing the API key on the preferences still allow me to use the extension, even though the key is wrong.
- I still don't know how to create a workflow in order to test your extension.

**Comment by olekristianbe** [comment]
Hi! Thanks for the feedback. I’ll make changes this weekend and get back to you. Have a great day!

**Pull request closed without merging**